topological_action_planner/README.md

```diff
@@ -0,0 +1,39 @@
+# topological_action_planner
+ROS package for planning over a topological navigation mesh. 
+Nodes are areas of interest in an ED world model and edges between these areas are labeled with the action to take to traverse the edge,
+    like opening a door or plain driving.
+
+## Try it out
+Run 
+```bash
+roslaunch topological_action_planner start.launch
+rosservice call /hero/topological_action_planner/get_plan  "origin:
+  entity: 'operator_table'
+  area: 'in_front_of'
+destination:
+  entity: 'dinner_table'
+  area: 'in_front_of'"
+```
+or 
+```bash
+rosservice call /hero/topological_action_planner/get_plan  "origin:
```

Comment:
Again, remove reference to hero

Reply:
Again, I want to be able to copy-paste for testing this easily. I'll add a note though